api/test/search.test.js

```diff
@@ -62,4 +63,16 @@ describe('search', () => {
       expect(name).toEqual('Human-GEM');
     }
   });
+
+  test('global search should return 50 results for multiple models', async () => {
```

Comment:
Maybe we should remove the number 50? Since we are not specifying a limit for global search (right?) and we are only testing `toBeGreaterThan(0)` the title is a bit confusing

Reply:
Yea this seems like a blunder, thank you for spotting it! Fixed in: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/892/commits/ffd0952393a62201f4ca49cec77f07d540276560